index.bs

```diff
@@ -308,6 +314,10 @@ interface XRDepthInformation {
   [SameObject] readonly attribute XRRigidTransform normDepthBufferFromNormView;
   readonly attribute float rawValueToMeters;
   readonly attribute XRView? view;
+
+  readonly attribute boolean isValid;
+  readonly attribute float depthNear;
```

Comment:
I think these maybe snuck in here from another change? Is it possible to keep them as a separate change (Issue for them is Issue 43, though I thought we'd come to a decision that maybe we didn't need to do this?) Per offline chats with Piotr, if we are going to merge them in we'd want some math on how to use them, and he thinks there may be additional spec text changes needed.

Reply:
In a previous discussion, I had mentioned that I would combine them in 1 PR.
Happy to pull it into separate ones if it's more clear.

Reply:
Per meeting this week, let's split so we can merge this, since I think more tweaks may be needed for that one.

Reply:
FYI, isValid is used by three.js. I removed it from the spec but will likely keep it on our implementation.

Reply:
Ack